2001-3000/LIT2053Nagara.xml

```diff
@@ -49,11 +51,25 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
          <change who="ES" when="2016-02-09">CREATED: text record</change>
          <change when="2017-02-14" who="ES">updated title keywords</change>
          <change when="2017-09-05" who="DR">Corrected title</change>
+         <change who="MKr" when="2022-08-29">Added summary and incipit based on the info provided by Sophia Dege-Müller</change>
       </revisionDesc>
    </teiHeader>
+   <text>
+        <body>
+          <div type="bibliography"/>
+
+          <div xml:lang="gez" type="edition">
+            <note>The incipit is taken Faitlovitch MS 13</note>
```

Comment:
Maybe add "from" - the incipit is taken "from" Faitlovitch...
what do you think?

Reply:
Oh and also from which folio is the incipit?